package.json

```diff
@@ -42,7 +42,7 @@
     "lib/**/*.d.ts"
   ],
   "devDependencies": {
-    "@biomejs/biome": "1.8.3",
+    "@biomejs/biome": "^1.8.3",
```

Comment:
Biome minor version can have new rules, additional fixes enabled between versions, they specifically recommend pinning its version to prevent unexpected breakage
<img width="742" alt="image" src="https://github.com/user-attachments/assets/b2a924ec-325c-4c95-9ae9-ac1309023ada">

Reply:
I think the version is already pinned using yarn, in `yarn.lock`.

Any update typically needs to pass CI, usually triggered via dependabot.

Reply:
If you have already installed dependencies then yes, but if for example you clone new project ant run npm install it will attempt to download never dependencies matching range, not sure on how specifically yarn handles it, but in this case I think it's better to pin it.